src/coffee/directives/api/utils/fit-helper.coffee

```diff
@@ -9,5 +9,6 @@ angular.module('uiGmapgoogle-maps.directives.api.utils')
           everSet = true unless everSet
           point = if _.isFunction markerOrPoint.getPosition then markerOrPoint.getPosition() else markerOrPoint
         bounds.extend point
-      gMap.fitBounds(bounds) if everSet
+      if everSet
+        $timeout () -> gMap.fitBounds(bounds)
```

Comment:
Yeah I def made changes here. https://github.com/nmccready/angular-google-maps/blob/dev/nmccready/issue_1458_polyline_fit/src/coffee/directives/api/utils/fit-helper.coffee

Anyway you can make a unit test to make sure your fix still works even after I merge the changes listed above.